docs/admin/prerequisites/nginx.rst

```diff
@@ -1,8 +1,10 @@
 .. _nginx:
 
-NGINX ingress configuration
+NGINX ingress
 ============================
 
+Renku provides some `kubernetes services <https://kubernetes.io/docs/concepts/services-networking/service/>`_ that need to be exposed. Kubernetes uses NGINX Ingress for managing routes and controlling traffic.
```

Comment:
it's not the kubernetes uses nginx for ingress by default, but we happen to provide a setup which assumes an nginx ingress. Maybe this should be reworded slightly to reflect this fact.

Reply:
:+1:  will rephrase, thanks!